src/config.ts

```diff
@@ -19,6 +19,10 @@ export interface BobConfig {
   commands?: {
     [cmdName: string]: Command;
   };
+  dists?: {
+    distDir: string;
+    distPath?: string;
+  }[]
```

Comment:
Do we need this or could we maybe have `cjs`, `esm`, `nodeEsm` flags? @ardatan

Reply:
what's the story behind `dists`?

Reply:
Having es5 builds for graphql-tools
https://github.com/ardatan/graphql-tools/blob/master/bob.config.js#L9
We have needed that before in MSTeams

Reply:
Yes it is actually. https://github.com/ardatan/graphql-tools/blob/master/package.json#L10
Here we run tsc twice in parallel, one is for modern es and the other one is for es5.

Reply:
I am not sure this old solution is best but it worked for MSTeams :D It was a long time ago, we can change it for sure. But that was the reason behind having this `dists` thing.

Reply:
@kamilkisiela @kamilkisiela (for me) add nodeEsm flag to the PR and merge it

Reply:
(also for me) update validation rules in bob validate to make sure it’s in sync